src/other/ReplayPage.jsx

```diff
@@ -140,12 +142,14 @@ const ReplayPage = () => {
       setIndex(0);
       const positions = await response.json();
       setPositions(positions);
+      setLoading(false);
```

Comment:
Let's call it once in a try-finally.

Reply:
Check how we do it in other places.

Reply:
sorry i had a moment. was trying to work out what to do with the error in catch, and then realised i could just do try>finally instead